### PR TITLE
Bump Go to 1.18

### PR DESCRIPTION
**Pull request by infraredgirl**

### CODE REVIEW

File: modules/k8s/config_test.go [hunk 1]
```diff
@@ -114,6 +114,7 @@ preferences: {}
 users:
 - name: minikube
   user:
+    as-user-extra: null
```
Comment (infraredgirl) [inline modules/k8s/config_test.go:117]: For some reason, without this change, five k8s tests fail. I don't really understand why, but I've confirmed that they fail when run with go 1.18 but pass with go 1.17. Very strange.

**Comment by infraredgirl** [comment]
The rest of the failing tests are failing on master as well.

**Review by denis256 — APPROVED** [review]
Will take a look on failing tests

**Comment by infraredgirl** [comment]
Thanks - going to merge this in.

**Pull request merged**